test/quantization/test_quantization.py

```diff
@@ -315,6 +315,29 @@ def checkQuantized(model):
 
         checkQuantized(model)
 
+    def test_normalization(self):
```

Comment:
This test (and other tests here) only test for FBGEMM, needs to be fixed in a later PR.